packages/documentation-site/patternfly-docs/content/design-guidelines/components/code-block/code-block.md

```diff
@@ -15,14 +19,18 @@ Code block has 4 main components.
 
 ### Multiple action buttons
 
-<img src="./img/code-block-multiple-actions.png" alt="Code block with multiple actions" width="355"/>
+<div class="ws-docs-content-img">
+![Example of a code block with multiple actions.](./img/code-block-multiple-actions.svg)
+</div>
 
 Code blocks can include multiple actions. Any actions are optional and customizable, but the copy action is the most common.
 
 The additional actions to the copy action can include the option to show a checkmark after the user has clicked the action button. The checkmark can be used to indicate to the user that the action was successfully triggered.
 
 ### Show more/less
 
-<img src="./img/code-block-collapsed-expanded.png" alt="Expanded and collapsed code block" width="758"/>
+<div class="ws-docs-content-img">
```

Comment:
```suggestion
<div class="ws-docs-content-img">
```

I'm also thinking that moving the image below the docs content here would maybe make more sense? thoughts?